### ISSUE TITLE
Question about PubMed performance

### ISSUE THREAD

**Issue by ltz0120** [CRIPAC-DIG/GRACE#7]
Hi,

In your paper, GRACE achieves 86.7% in Pubmed, and DGI achieves 86% in Pubmed. However, in the DGI paper, the performance of DGI only achieves 76.8% in Pubmed. I also notice you follow the DGI setting in your experiments. How did you make an improvement of almost 10% on DGI?

**Comment by downeykking** [comment]
The dataset spilt setting is different. DGI uses public spilt while GRACE uses 10% train, 10% valid, and 80% test random spilt.

**Comment by qingninggo** [comment]
> The dataset spilt setting is different. DGI uses public spilt while GRACE uses 10% train, 10% valid, and 80% test random spilt.

 GRACE  train/test split follows a 90% / 10% .  Isn't that it?

**Comment by downeykking** [comment]
> 

In paper,  
> On these citation networks, we randomly select 10% of the nodes as the training set, 10% nodes as the validation set, and leave the rest nodes as the test set.

train/valid/test split follows a 10%/10%/80%. 

And in code, train/test split follows a 10%/90%.
https://github.com/CRIPAC-DIG/GRACE/blob/master/train.py#L41C4
https://github.com/CRIPAC-DIG/GRACE/blob/master/eval.py#L58

**Comment by downeykking** [comment]
> > The dataset spilt setting is different. DGI uses public spilt while GRACE uses 10% train, 10% valid, and 80% test random spilt.
> 
> GRACE train/test split follows a 90% / 10% . Isn't that it?

In paper,  
> On these citation networks, we randomly select 10% of the nodes as the training set, 10% nodes as the validation set, and leave the rest nodes as the test set.

train/valid/test split follows a 10%/10%/80%. 

And in code, train/test split follows a 10%/90%.
https://github.com/CRIPAC-DIG/GRACE/blob/master/train.py#L41C4
https://github.com/CRIPAC-DIG/GRACE/blob/master/eval.py#L58

**Comment by qingninggo** [comment]
> > > The dataset spilt setting is different. DGI uses public spilt while GRACE uses 10% train, 10% valid, and 80% test random spilt.
> > 
> > 
> > GRACE train/test split follows a 90% / 10% . Isn't that it?
> 
> In paper,
> 
> > On these citation networks, we randomly select 10% of the nodes as the training set, 10% nodes as the validation set, and leave the rest nodes as the test set.
> 
> train/valid/test split follows a 10%/10%/80%.
> 
> And in code, train/test split follows a 10%/90%. https://github.com/CRIPAC-DIG/GRACE/blob/master/train.py#L41C4 https://github.com/CRIPAC-DIG/GRACE/blob/master/eval.py#L58

Thanks for answering! So, no validation sets are used in the code?

**Comment by qingninggo** [comment]
> > > > The dataset spilt setting is different. DGI uses public spilt while GRACE uses 10% train, 10% valid, and 80% test random spilt.
> > > 
> > > 
> > > GRACE train/test split follows a 90% / 10% . Isn't that it?
> > 
> > 
> > In paper,
> > > On these citation networks, we randomly select 10% of the nodes as the training set, 10% nodes as the validation set, and leave the rest nodes as the test set.
> > 
> > 
> > train/valid/test split follows a 10%/10%/80%.
> > And in code, train/test split follows a 10%/90%. https://github.com/CRIPAC-DIG/GRACE/blob/master/train.py#L41C4 https://github.com/CRIPAC-DIG/GRACE/blob/master/eval.py#L58
> 
> Thanks for answering! So, no validation sets are used in the code?

GCA？Are validation sets used in GCA?

**Comment by downeykking** [comment]
> > > > > The dataset spilt setting is different. DGI uses public spilt while GRACE uses 10% train, 10% valid, and 80% test random spilt.
> > > > 
> > > > 
> > > > GRACE train/test split follows a 90% / 10% . Isn't that it?
> > > 
> > > 
> > > In paper,
> > > > On these citation networks, we randomly select 10% of the nodes as the training set, 10% nodes as the validation set, and leave the rest nodes as the test set.
> > > 
> > > 
> > > train/valid/test split follows a 10%/10%/80%.
> > > And in code, train/test split follows a 10%/90%. https://github.com/CRIPAC-DIG/GRACE/blob/master/train.py#L41C4 https://github.com/CRIPAC-DIG/GRACE/blob/master/eval.py#L58
> > 
> > 
> > Thanks for answering! So, no validation sets are used in the code?
> 
> GCA？Are validation sets used in GCA?

GCA utilizes validation sets, whereas GRACE does not incorporate validation sets. You can refer to their `eval.py`.
